GoogleForCreators/web-stories-wp
# Redesign Page Header

## Context



## Summary
Updates the action buttons in the page header.


## Relevant Technical Choices



## To-do

- [x] Missing icon for "Switch to Draft"



## User-facing changes
Before:
<img width="336" alt="Screenshot 2021-02-09 at 14 14 44" src="https://user-images.githubusercontent.com/3294597/107362285-d7be8680-6ad8-11eb-8340-13042af0999a.png">

After:
<img width="210" alt="Screenshot 2021-03-05 at 21 00 09" src="https://user-images.githubusercontent.com/3294597/110161295-70e97000-7ded-11eb-8ce3-6adcc672882e.png">




## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Open a new story
2. Verify the page header buttons look as in [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=6515%3A521262)
3. Publish the story and verify there is "Save to Draft" button now instead of "Save draft" button.
4. Verify that during saving, a spinner is displayed around the save draft/switch to draft button.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6205
